omniledger/service/service_test.go

```diff
@@ -389,6 +458,33 @@ func (s *ser) service() *Service {
 	return s.services[0]
 }
 
+// caller gives us a darc, and we try to make an evolution request.
+func (s *ser) testDarcEvolution(t *testing.T, d2 darc.Darc) GetProofResponse {
+	ctx := darcToCtx(t, d2, s.signer)
+	_, err := s.service().AddTransaction(&AddTxRequest{
+		Version:     CurrentVersion,
+		SkipchainID: s.sb.SkipChainID(),
+		Transaction: ctx,
+	})
+	require.Nil(t, err)
+
+	// wait for the block to be processed (it should fail due to the wrong
```

Comment:
It should fail? This test seems to be checking for success?

Reply:
wrong comment, but this function doesn't do any real assertion on the response, the tests do it.